scripts/pull_request.sh

```diff
@@ -5,19 +5,17 @@
 set -e
 set -o pipefail
 
-################################################################################
-# Global Variables (we can't use GITHUB_ prefix)
-################################################################################
-
-API_VERSION=v3
-BASE=https://api.github.com
-AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
-HEADER="Accept: application/vnd.github.${API_VERSION}+json"
-HEADER="${HEADER}; application/vnd.github.antiope-preview+json; application/vnd.github.shadow-cat-preview+json"
+make_curl_config() {
+    tmpdir="$(umask 077 && mktemp -d)"
+    curl_config="${tmpdir}/curl_config"
+    cat >"${curl_config}" <<EOF
+--user "${GITHUB_ACTOR}"
+-H "Authorization: token ${GITHUB_TOKEN}"
+-H "Accept: application/vnd.github.v3+json, application/vnd.github.antiope-preview+json, application/vnd.github.shadow-cat-preview+json"
```

Comment:
This header is really important because GitHub is deprecating GET params for the user token. This was never an issue here, but it's on my mind because I've been updating older scripts.

Reply:
I’m not sure that I understand; did I port the header wrong? I was a bit
confused by the original `Accept: foo; bar` syntax; I’m used to seeing
comma-separated values, so I used those here.